src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -88,14 +88,6 @@ abstract class FixupSession(
         } catch (x: Exception) {
           logger.debug("Session cleanup error", x)
         }
-
-        runInEdt {
-          try { // Disposing inlay requires EDT.
-            Disposer.dispose(this)
```

Comment:
the sessions calls `cancelationToken.dispose()` now

Reply:
Cool!